### ISSUE TITLE
Search tools icon missing alt text 

### ISSUE THREAD

**Issue by camillevilla** [sul-dlss/sul-bento-app#284]
Add `aria-hidden=”true” alt=””` to 
`<img class="media-object card-img-left" src="/assets/search-461e62f7d72617eceb6fdf7c8e03337846517b0e8e526817d75e4682a794a389.svg">`
![homepageM](https://user-images.githubusercontent.com/5402927/88758620-b63e1480-d11d-11ea-879f-f3d27d346760.png)
